src/components/Footer/FooterNav.tsx

```diff
@@ -0,0 +1,165 @@
+import { NavItem, RequireOnlyOne } from '../../types';
+import Link from '../Link';
+import Accordion, { AccordionPanel } from '../Accordion';
+
+export type NavItemFooter = RequireOnlyOne<NavItem, 'to' | 'href'>;
+
+export interface NavItemsFooter {
+  title?: string;
+  navItems: NavItemFooter[];
+  altTo?: string;
+  altHref?: string;
+  altLinkText?: string;
+}
+
+export interface NavItemsInternal extends NavItemsFooter {
+  id: string;
+}
+
+interface Props {
+  navItems: NavItemsFooter[];
+}
+
+const FooterNav = (props: Props) => {
+  const isLarge = (): boolean => {
+    const fontSize = parseFloat(getComputedStyle(document.documentElement).fontSize);
+    return window.innerWidth >= fontSize * 48;
+  };
+
+  const [items, setItems] = useState<NavItemsInternal[]>([]);
+  const [columns, setColumns] = useState<NavItemsInternal[][]>([]);
+  const [isViewportSmall, setIsViewportSmall] = useState(!isLarge());
+
+  const footerNavId = useId();
+
+  useEffect(() => {
+    const onResize = () => {
+      if (isLarge()) {
+        if (isViewportSmall) {
+          setIsViewportSmall(false);
+        }
+      } else if (!isViewportSmall) {
+        setIsViewportSmall(true);
+      }
+    };
+
+    window.addEventListener('resize', onResize);
+    return () => {
+      window.removeEventListener('resize', onResize);
+    };
+  }, [isViewportSmall]);
+
+  useEffect(() => {
+    if (items.some((item) => item.title) && !items.every((item) => item.title)) {
+      console.error('Footer nav items must either all have titles, or none have titles.');
+    }
+  }, [items]);
+
+  useEffect(() => {
+    setItems(
+      props.navItems.map((item, i) => {
+        return {
+          ...item,
+          id: footerNavId + i
+        };
+      })
+    );
+  }, [footerNavId, props.navItems]);
+
+  useEffect(() => {
+    const colCount = 3;
+    const columnSize = Math.floor(items.length / colCount);
+    let remainder = items.length % colCount;
+
+    const originalItems = [...items];
+    const cols: NavItemsInternal[][] = [];
+    for (let col = 0; col < colCount; col++) {
+      const itemCount = remainder > 0 ? columnSize + 1 : columnSize;
+      const colItems: NavItemsInternal[] = [];
+      for (let i = itemCount; i > 0; i--) {
+        const nextItem = originalItems.shift();
+        if (nextItem) {
+          colItems.push(nextItem);
+        }
+      }
+      cols.push(colItems);
+      remainder--;
+    }
+    setColumns(cols);
+  }, [items]);
+
+  return (
+    <nav className="bsds-nav-footer">
+      <ListManager isViewportSmall={isViewportSmall} items={items} columns={columns} />
+    </nav>
+  );
+};
+
+interface ListManagerProps {
+  isViewportSmall: boolean;
+  items: NavItemsInternal[];
+  columns: NavItemsInternal[][];
+}
+
+const ListManager = (props: ListManagerProps) => {
+  if (props.isViewportSmall && props.items.every((item) => item.title)) {
+    return (
+      <Accordion headingLevel="h2">
```

Comment:
when i tab through on xs, i'm losing focus between accordions

Reply:
This seems to be an accordion bug. Should we make a separate ticket?